applications/rag/frontend/main.tf

```diff
@@ -80,9 +80,6 @@ resource "kubernetes_service" "rag_frontend_service" {
   metadata {
     name      = "rag-frontend"
     namespace = var.namespace
-    annotations = var.add_auth ? {
```

Comment:
Why did we delete this?

Reply:
Discussed with Umesh, we moved all annotation operation back to IAP